devolpment_shortcut.py

```diff
@@ -0,0 +1,111 @@
+"""
+Provide access to data retrival from degreeworks using tokens as well as data parsing methods.
+
+Uses selenium to retrive data and beautifulSoup to parse it.
+"""
+import time
+import json
+import os
+import pickle
+from selenium import webdriver
+from selenium.webdriver.chrome.options import Options
+from selenium.webdriver.chrome.service import Service
+from webdriver_manager.chrome import ChromeDriverManager
+from bs4 import BeautifulSoup
+
+# import privateInfo
+button = False
+
+

```

Comment:
this needs click formatting like in manual_get_data, the inputs should be passed through the command line